components/sup/src/manager/mod.rs

```diff
@@ -463,6 +466,21 @@ impl Manager {
             }
         };
 
+        if feat::is_enabled(feat::InstallHook) {
+            if let Ok(package) =
+                PackageInstall::load(&service.pkg.ident, Some(Path::new(&*FS_ROOT_PATH)))
+            {
+                if let Err(err) = common::command::package::install::check_install_hooks(
```

Comment:
`--ignore-install-hook` is intentionally only available on `hab pkg install` and not on `hab svc load`. In a supervisor context any install hook should be run if provided otherwise It is safe to assume the service would fail.

Definitely agree on the function name. I never felt great about it and I like your suggested name.